js/preview/listblock.scss

```diff
@@ -2,21 +2,26 @@
     position: relative;
     height: 300px;
     padding: 20px 40px;
-    & > div:not([data-jsx-placeholder]){
+    max-width: 600px;
+    margin: 0 auto;
+    & > div:not([data-jsx-placeholder]), & > img{
         width: calc(100% - 40px);
         max-width: 550px;
-        height: 300px;
+        height: calc(300px - 2rem);
```

Comment:
gadsie? wat is dit voor iets weirds?

Reply:
Dit is om te compenseren met de padding die erop zit